Sources/Sentry/SentrySDK.m

```diff
@@ -153,6 +158,99 @@ + (void)setStartTimestamp:(NSDate *)value
     startTimestamp = value;
 }
 
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+SentrySamplingContext *
+samplingContextForAppLaunches(void)
+{
+    SentryTransactionContext *transactionContext =
+        [[SentryTransactionContext alloc] initWithName:@"app.launch" operation:@"profile"];
+    transactionContext.forNextAppLaunch = YES;
```

Comment:
We considered that approach as well but requiring string comparisons is worse, because it's a more heavyweight task, makes readability worse at the callsite, and throws out compile time safety by using an arbitrary domain in string values.

Reply:
Thanks for the clarification. We can also keep it.